LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 shaheryar abid
```

Comment:
The name on this Copyright should display your name. Kindly take a look at this to make appropriate corrections.